tools/code_format/check_format.py

```diff
@@ -65,7 +65,9 @@
 # perform temporary registrations.
 REGISTER_FACTORY_TEST_ALLOWLIST = (
     "./test/common/config/registry_test.cc", "./test/integration/clusters/",
-    "./test/integration/filters/")
+    "./test/integration/filters/",
+    "./test/extensions/http/header_formatters/preserve_case/preserve_case_formatter_integration_test.cc"
```

Comment:
The comment above suggests you should be using `InjectFactory` instead of of REGISTER_FACTORY, thoughts?

Reply:
This isn't possible for integration tests. This is here for the same reason `test/integration/filters` is here.

Reply:
Huh I thought I'd seen InjectFactory used in integration tests, but I could be wrong here

Reply:
Ah OK it looks like it can. Sure I can fix that.